docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -131,6 +131,29 @@ each vulnerability. The rating indicates how important an update is:
 | **Moderate**  | Releases of moderate severity should be applied as soon as possible. They allow the unauthorized editing or creation of content. *Examples: Cross Site Scripting (XSS) in template helpers.* |
 | **Low**       | Low risk releases fix information disclosure and read-only privilege escalation vulnerabilities. These updates should also be applied as soon as possible, but with an impact-dependent priority. *Example: Exposure of the core version number, Cross Site Scripting (XSS) limited to the admin interface.* |
 
+### Internal Security Process
+
+Follow these instructions in sequence as much as possible:
+
+ * When receiving a report:
+   * Perform initial criticality assessment
+   * Assign a unique identifier (see "Acknowledgement and disclosure")
+   * Respond to issue reporter with this identifier on the same discussion thread (cc security@silverstripe.org). Clarify issue if required.
+   * If encrypted information is provided, add pass phrases into the SilverStripe Ltd. LastPass account. Keep encrypted documents in Google Drive and only share directly with relevant participants
+   * Create a draft page under [Open Source > Download > Security Releases](https://www.silverstripe.org/admin/pages/edit/show/794) on silverstripe.org. Describe the issue in a readable way, make the impact clear. Credit the author if applicable. 
+   * Add a new bug on our [Open Source Security JIRA board](https://silverstripe.atlassian.net/secure/RapidBoard.jspa?rapidView=198&view=detail). Add a link to the [Google Groups](https://groups.google.com/a/silverstripe.com/forum/#!forum/security) discussion thread so it's easy to review follow up messages.
+   * Clarify who picks up owns the issue resolution
+ * When developing a fix:
+   * Add fixes on the [http://github.com/silverstripe-security](http://github.com/silverstripe-security) repo
+   * Get them peer reviewed by posting on security@silverstripe.org with a link to github
```

Comment:
We need to add these links to the internal jira; Keep the jira as the central point of management for these issues rather than the mailing list (which you can post to too, but we should be able to see all progress made on the jira ticket).